dapp/src/utils.ts

```diff
@@ -18,7 +18,7 @@ export const signClaim = async (
 };
 
 export const getMichelineStringBytes = (str: string) => {
-  const strLenght = str.length.toString(16).padStart(8, '0');
-  const bytes = `0501${strLenght}${char2Bytes(str)}`;
+  const strLength = ((char2Bytes(str).length)/2).toString(16).padStart(8, '0');
```

Comment:
You should put the result of `char2Bytes(str)` in a variable to avoid duplicating it. It's not so much about performance but to make future changes safer.

Reply:
Done